src/nunavut/lang/cpp/__init__.py

```diff
@@ -923,16 +923,41 @@ def filter_destructor_name(language: Language, instance: pydsdl.Any) -> str:
     return "~" + "<".join(declaration_parts)
 
 
+@template_language_filter(__name__)
+def filter_explicit_decorator(language: Language, instance: pydsdl.Any, special_method: SpecialMethod) -> str:
+    """
+    Emit the constructor name, decorated with "explicit" if it has only one argument
```

Comment:
"...other than a leading allocator." <- right? Or am I not reading this code correctly?

Reply:
If it has a leading allocator then there will be at least two arguments for the initializing constructor, in which case "explicit" is not needed.

In looking through the leading-allocator support I discovered it's kinda broken.  Created this issue to track it.  Too much to do here as it looks like a bigger lift to do it right.
https://github.com/OpenCyphal/nunavut/pull/320